src/hypothesis/strategies.py

```diff
@@ -176,19 +177,8 @@ def one_of(*args):
             args = tuple(args[0])
         except TypeError:
             pass
-
-    strategies = []
```

Comment:
The reason for these moves is so that that the `is\_empty` checks only happen once the tests start running, so that you get correct behaviour if you have a `deferred()` here.